nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/DocumentBlobManager.java

```diff
@@ -85,6 +86,46 @@
      */
     void notifyChanges(Document doc, Set<String> xpaths);
 
+    /**
+     * Notifies the blob manager that the document was made a record.
+     *
+     * @param doc the document
+     * @since 11.1
+     */
+    void notifyMakeRecord(Document doc);
```

Comment:
Add default implementation doing nothing?

Reply:
Here I'd rather not do it, the component/service has only one possible implementation in Nuxeo today.